### ISSUE TITLE
INT8 No implementation problems 

### ISSUE THREAD

**Issue by steven12356789** [onnx/onnx#3381]
HI, i use ResneSt model from the following link.
https://github.com/rwightman/pytorch-image-models
Now I can transform the model to ONNX, but during transform to INT8 will meet this warning.
How to solve this problem?

onnx 1.7.0
onnxruntime 1.5.2
tensorrt 7.2.1.4
torch 1.7.0

WARN TRT: No implementation of layer (Unnamed Layer* 69) [Shuffle] + Transpose_52 obeys the requested constraints in strict mode. No conforming implementation was found i.e. requested layer computation precision and output precision types are ignored, using the fastest implementation. trt_utils.cpp:253
WARN TRT: No implementation of layer ReduceSum_68 obeys the requested constraints in strict mode. No conforming implementation was found i.e requested layer computation precision and output precision types are ignored, using the fastest implementation. trt_utils.cpp:253
WARN TRT: No implementation of layer ReduceSum_103 obeys the requested constraints in strict mode. No conforming implementation was found i.e requested layer computation precision and output precision types are ignored, using the fastest implementation. trt_utils.cpp:253
WARN TRT: No implementation of layer (Unnamed Layer* 158) [Shuffle] obeys the requested constraints in strict mode. No conforming implementa
WARN TRT: No implementation of layer Softmax_116 obeys the requested constraints in strict mode. No conforming implementation was found i.e requested layer computation precision and output precision types are ignored, using the fastest implementation. trt_utils.cpp:253
WARN TRT: No implementation of layer (Unnamed Layer* 160) [Shuffle] + Transpose_117 obeys the requested constraints in strict mode. No conforming implementation was found i.e requested layer computation precision and output precision types are ignored, using the fastest implementation. trt_utils.cpp:253
WARN TRT: No implementation of layer ReduceSum_133 obeys the requested constraints in strict mode. No conforming implementation was found i.e requested layer computation precision and output precision types are ignored, using the fastest implementation. trt_utils.cpp:253
WARN TRT: No implementation of layer ReduceSum_166 obeys the requested constraints in strict mode. No conforming implementation was found i.e requested layer computation precision and output precision types are ignored, using the fastest implementation. trt_utils.cpp:253
WARN TRT: No implementation of layer (Unnamed Layer* 247) [Shuffle] obeys the requested constraints in strict mode. No conforming implementation was found i.e requested layer computation precision and output precision types are ignored, using the fastest implementation. trt_utils.cpp:253
WARN TRT: No implementation of layer Softmax_179 obeys the requested constraints in strict mode. No conforming implementation was found i.e requested layer computation precision and output precision types are ignored, using the fastest implementation. trt_utils.cpp:253
WARN TRT: No implementation of layer (Unnamed Layer* 249) [Shuffle] + Transpose_180 obeys the requested constraints in strict mode. No conforming implementation was found i.e requested layer computation precision and output precision types are ignored, using the fastest implementation. trt_utils.cpp:253
WARN TRT: No implementation of layer ReduceSum_196 obeys the requested constraints in strict mode. No conforming implementation was found i.e requested layer computation precision and output precision types are ignored, using the fastest implementation. trt_utils.cpp:253
WARN TRT: No implementation of layer ReduceSum_229 obeys the requested constraints in strict mode. No conforming implementation was found i.e requested layer computation precision and output precision types are ignored, using the fastest implementation. trt_utils.cpp:253

**Comment by askhade** [comment]
@steven12356789 : at what stage did you see these warnings? During conversion to onnx or during inferencing? If this is during conversion then please open this issue in pyotrch repo and if this is during inference then please open it in onnxruntime repo

**Comment by steven12356789** [comment]
Thanks for your response.
I faced this problem when I wanted to use tensorRT to speed up inference and use C++ to convert fp32 to int8.
